recipes/setupmeta/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Looks like the license file is already in the PyPI tarball, so this file can be removed from here.